components/camel-aws/src/main/java/org/apache/camel/component/aws/sqs/SqsConfiguration.java

```diff
@@ -83,6 +87,18 @@
     private String redrivePolicy;
 
     /**
+     *  Whether or not the queue is a FIFO queue
+     */
+    public boolean isFifoQueue() {
+        // AWS docs suggest this is valid derivation.
+        // FIFO queue names must end with .fifo, and standard queues cannot
+        if (queueName.endsWith(".fifo")) {
```

Comment:
Please create a constant and also put it on the left side of the comparison. Thanks.

Reply:
I don't think a constant would add any value here.  Also note that this is not an `equals()` comparison, and queueName cannot be null, as this is validated in `createEndpoint()`

Reply:
Makes sense. Thanks.